lib/vrf.h

```diff
@@ -62,6 +59,7 @@ struct vrf_data {
 			uint32_t table_id;
 		} l;
 	};
+	char ns_name[NS_NAMSIZ];
```

Comment:
Please put this under the `l` structure (which is meant for Linux-specific VRF data). And if possible please rename it to `netns_name` to be more concise.

Reply:
ok